tests/fixtures/recipes/echo-recipe.yml

```diff
@@ -0,0 +1,8 @@
+# Example recipe designed solely for use with the custom echo backend
```

Comment:
I would prefer the file name `echo_recipe.yml` than `echo-recipe.yml`.
Because custom files in `tests/fixtures/custom/` is using underscore for the delimiter.